### ISSUE TITLE
Possibly unintentional missing `override` for `ProcessCandidateTabStopOverride`

### ISSUE THREAD

**Issue by Youssef1313** [unoplatform/uno#10958]
### Current behavior

https://github.com/unoplatform/uno/blob/e0fe6df045d9d90599bca201032a95e637fca6d4/src/Uno.UI/UI/Xaml/Controls/CalendarView/CalendarView_Partial_Interaction.cs#L372

unused private method

### Expected behavior

Is this intended to override this method?

https://github.com/unoplatform/uno/blob/e0fe6df045d9d90599bca201032a95e637fca6d4/src/Uno.UI/UI/Xaml/UIElement.mux.cs#L356

### Works on UWP/WinUI

None
